docs/eventlisteners.md

```diff
@@ -632,6 +632,17 @@ spec:
     value: $(body.pull_request.head.short_sha)
 ```
 
+## Response from the EventListener
```

Comment:
Why is not getting an anchor at the top like others?
https://github.com/tektoncd/triggers/blob/445d9a434e4021118f32bcef01347c829e5daa11/docs/eventlisteners.md